bin/runner.js

```diff
@@ -2,25 +2,21 @@
  * Module dependencies
  */
 const Nightwatch = require('../lib/index.js');
-const {Logger, shouldReplaceStack, alwaysDisplayError} = require('../lib/utils');
-const path = require('path');
+const {
+  Logger,
+  shouldReplaceStack,
+  alwaysDisplayError,
+  loadTSNode,
+  findTSConfigFile
+} = require('../lib/utils');
 
 try {
-  const projectTsFile = path.join(process.cwd(), 'nightwatch', 'tsconfig.json');
-  let projectTsExists = false;
-  try {
-    require(projectTsFile);
-    projectTsExists = true;
-
-    require('ts-node').register({
-      esm: false,
-      project: projectTsFile
-    });
-  } catch (err) {
-    // eslint-disable-line
-  }
-
-  Nightwatch.cli(function(argv) {
+  Nightwatch.cli(function (argv) {
+    const projectTsFilePath = findTSConfigFile();
+    if (projectTsFilePath !== '') {
```

Comment:
I actually missed this before, but we also need to check for the `disable_typescript` setting, which might mean that we need to move all this into the CLIRunner class after we parse the settings.